openmdao/drivers/tests/test_differential_evolution_driver.py

```diff
@@ -0,0 +1,1052 @@
+""" Unit tests for the DifferentialEvolutionDriver Driver."""
+
+import unittest
+import os
+
+import numpy as np
+
+import openmdao.api as om
+from openmdao.drivers.differential_evolution_driver import DifferentialEvolution
+from openmdao.test_suite.components.branin import Branin, BraninDiscrete
+from openmdao.test_suite.components.paraboloid import Paraboloid
+from openmdao.test_suite.components.paraboloid_distributed import DistParab
+from openmdao.test_suite.components.sellar_feature import SellarMDA
+from openmdao.test_suite.components.three_bar_truss import ThreeBarTruss
+
+from openmdao.utils.assert_utils import assert_near_equal
+from openmdao.utils.mpi import MPI
+
+try:
+    from openmdao.vectors.petsc_vector import PETScVector
+except ImportError:
+    PETScVector = None
+
+extra_prints = False  # enable printing results
+
+class TestDifferentialEvolution(unittest.TestCase):
+    def setUp(self):
+        import openmdao.api as om
+        import numpy as np
+        import os
+        os.environ['DifferentialEvolutionDriver_seed'] = '11'  # make RNG repeatable
+
+    def test_rastrigin(self):
+        ORDER = 6  # dimension of problem
+        span = 5   # upper and lower limits
+        extra_prints = False
```

Comment:
Get rid of all the "extra prints" stuff in the examples that are in the docs.  It will just confuse people.  (I think it is just this example.  The other one doesn't have them.)